ros_ws/src/teleoperation/launch/remote_control.launch

```diff
@@ -11,6 +11,7 @@ Launches all necessary nodes that let the user control the car with the keyboard
       type="joystick_controller"
       name="joystick_controller"
       output="log" >
+      <param name="invert_steering" type="bool" value ="true" />
```

Comment:
`<param>` is used to hard-code parameters which is probably not what we want here. A better way would be to use `<arg>` here:
```suggestion
       <arg name="invert_steering" default ="true" />
```